front_end/ui/legacy/RemoteDebuggingTerminatedScreen.ts

```diff
@@ -102,13 +102,16 @@ export class RemoteDebuggingTerminatedScreen extends VBox {
     );
   }
 
-  static show(reason: string): void {
+  static show(
+    uiMessage: string,
+    connectionLostDetails: {reason: string, code: string, errorType: string} = {reason: '', code: '', errorType: ''}
```

Comment:
Does it have to be initialized with empty strings? I thought `reason` is always present, even if it is just empty string. Other fields may be optional, but do we actually need to set them to empty strings? I think you can have optional fields in event payloads in `RNPerfMetrics`

Reply:
You are right. I've made all these fields optional here and in the corresponding diff D71821154.